### PR TITLE
Redis store to rate limit

### PR DESCRIPTION
**Pull request by ChaituVR**
- Add `RATE_LIMIT_DATABASE_URL` env

### CODE REVIEW

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -72,5 +72,5 @@ export function getIp(req) {
     ''
   ).split(',');
 
-  return ips[0].trim();
+  return sha256(ips[0].trim()).substring(0, 10);
```
Comment (wa0x6e) [inline src/helpers/utils.ts:75]: You should hash after retrieving that value. Editing the returned value is breaking current implementation

Comment (ChaituVR) [reply src/helpers/utils.ts:75]: Why 🤔 where are only using this as a key right 🤔

Comment (ChaituVR) [reply src/helpers/utils.ts:75]: Which one is breaking? how can i reproduce?

Comment (wa0x6e) [reply src/helpers/utils.ts:75]: Test are breaking. I think shutter is also using the unhashed IP

Comment (bonustrack) [reply src/helpers/utils.ts:75]: Good point, let's keep the function getIp unchanged and hash later in rateLimit file

Comment (bonustrack) [reply src/helpers/utils.ts:75]: We should do the same on score-api and hub

**Review by bonustrack — APPROVED** [review]
utACK

**Pull request merged**